src/app/menu-bar/menu-bar.component.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>TaskManager</title>
+  <meta charset="utf-8">
+</head>
+
+<body>
+  <nav class="navbar navbar-inverse">
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="#">TaskManager</a>
+      </div>
+      <ul class="nav navbar-nav">
+        <li class="active">
+          <a href="#">Home</a>
+        </li>
+        <li class="dropdown">
+          <a class="dropdown-toggle" data-toggle="dropdown" href="#">Home
+            <span class="caret"></span>
+          </a>
+          <ul class="dropdown-menu">
+            <li>
+              <a href="#">Func1</a>
+            </li>
+            <li>
+              <a href="#">Func2</a>
+            </li>
+            <li>
+              <a href="#">Func3</a>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="#">MyList</a>
```

Comment:
Please create appropriate menu items as we discussed

Reply:
Created